src/buffer/out/sgrStack.hpp

```diff
@@ -0,0 +1,238 @@
+/*++
```

Comment:
I do not like this being header only. I feel like the definition of the object should be in a .cpp file.

Reply:
Done.